pages/support.js

```diff
@@ -6,9 +6,19 @@ import { Footer } from "../components/Layout/Footer";
 import { ReusableButtons } from "../components/Support/ReusableButtons";
 import ScrollButton from "../components/Layout/ScrollButton";
 import { Resources } from "../components/Support/Resources";
-import resourcesData from "../database/SupportResources";
-
-export default function Support() {
+import { sanityClient } from "../lib/sanity";
+
+const supportQuery = `*[_type == "support"]{
+  _id, 
+  title, 
+  url, 
+  allyship, 
+  category, 
+  description, 
+  image
+  }`;
```

Comment:
I think it's fine for the order to be different from the original, but we can also add to the query what to [order](https://www.sanity.io/docs/query-cheat-sheet#b5aec96cf56c) the result by